workspace.code-workspace

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Change looks good. Can I ask though why commit this file? I'm not familiar with VSCode workspaces, what's the intended effect? It doesn't seem to apply any settings other than declaring a workspace for the entire project. Just for my understanding 🙂